packages/braid-design-system/src/lib/components/private/touchable/virtualTouchableRules.ts

```diff
@@ -1,9 +1,15 @@
 import { hitArea } from './hitArea';
 
+/*
+These rules are shared between locations that cannot share a common class
+e.g. between a pseudo element and a regular dom element
+*/
 export const virtualTouchableRules = {
-  transform: 'translateY(-50%)',
   minHeight: hitArea,
   minWidth: hitArea,
   height: '100%',
+  width: '100%',
+  transform: 'translate(-50%, -50%)',
   top: '50%',
+  left: '50%',
```

Comment:
Do we have/want a screenshot test for this?

Reply:
Thats an interesting one actually. Might have a look at the idea of having a screenshot of the implementation in the components themselves.

Reply:
Something like this:

![Screenshot 2024-05-16 at 1 55 04 PM](https://github.com/seek-oss/braid-design-system/assets/912060/6112d87e-d886-457e-9277-e4e4767913a5)

Reply:
I like it 👍

Reply:
Worth doing in this PR, or separately?

Reply:
I'll add to this one if you're happy?

Reply:
Go for it.